docker/CMSRucioClient/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM centos:7
+FROM almalinux:9
+
+ARG RUCIO_VERSION
 
 RUN yum upgrade -y && \
```

Comment:
To be really compliant yum is replaced with dnf in alma9. As I understand it they are different programs but generally the interface is the same so s/yum/dnf/g should work.